### DISCUSSION TITLE
Tailwind colours should be included in Filament's CSS by default

### DISCUSSION THREAD

**Discussion by pekka** [filamentphp/filament#6866] [Ideas]
Filament doesn't seem to package all Tailwind classes but only those which its UI needs, which is obviously fair and correct.

However, one customization use case that MUST be very common, and should be achievable without having to through a complex process, is colours. Say when customizing label colours in a `BadgeColumn`. It seems like a badge can only have the colour that Filament is embedding by default and anything else needs to be loaded additionally - a process that I haven't quite figured out how to do yet. (Working on it!)

tl;dr: TailwindCSS's colour classes should all be embedded by default even though Filament does not make use of all of them.

**Comment by danharrin** [comment]
In v3 we have built support for custom colors like this. However, we still do not compile them all into the CSS file, that would be huge. We use CSS variables.
